src/wrappers/any.jl

```diff
@@ -124,14 +124,20 @@ function Base.take!(buffer::CxxWrap.CxxWrapCore.SmartPointer{<:Buffer})
     return vec
 end
 
-function task_executor()
+function task_executor(ray_function)
     @info "task_executor called"
-    return getpid()
+    fd = GetFunctionDescriptor(ray_function)
+    func = eval(@__MODULE__, Meta.parse(CallString(fd)))
```

Comment:
Probably makes more sense to use `Main` here. Needs to be some extra thought

Reply:
yeah I think that's basically right...this is the callback that the (worker) coreworker calls when a task is scheduled to execute on that worker, which hands control over to the julia runtime to actually do the work.